lib/install/management/appliance.go

```diff
@@ -168,7 +168,14 @@ func (d *Dispatcher) deleteVM(vm *vm.VirtualMachine, force bool) error {
 	})
 	if err != nil {
 		err = errors.Errorf("Failed to destroy vm %q: %s", vm.Reference(), err)
-		return err
+		// Unregister then do we still want to deleteDatastoreFiles? something about attached volumes
```

Comment:
Attached volumes are ok here, cause only for vm delete, we'll delete any attached volumes. 
Delete datastore files will not delete volumes cause they are in different datastore and directories